modules/ob-creating-a-s2i-build.adoc

```diff
@@ -101,9 +102,9 @@ EOF
 .Example: Using `shp` CLI
 [source,terminal]
 ----
-$ shp build run s2i-nodejs-build --follow <1>
+$ shp build run s2i-nodejs-build --follow
 ----
-<1> Optional: By using the `--follow` flag, you can view the build logs in the output result.
```

Comment:
See the comment above. A code callout is the way to describe the flag here. Also, you specified that flag is optional above. That makes sense. I recommend keeping it optional here for consistency.